### ISSUE TITLE
Add support for building on Alpine targets (musl)

### ISSUE THREAD

**Issue by Elinvynia** [cobalt-org/cobalt.rs#763]
Hello! I have been messing around with trying to get the binary build statically linked, and there are two major roadblocks:

On: rust:alpine
`darling_macro` will refuse to compile while building on alpine with a musl target.

On: clux/muslrust
`sass-rs` will refuse to compile while building on debian with a musl target.

I've looked into https://github.com/compass-rs/sass-rs/issues/30 and tried tweaking various other things, but nothing worked.

Thanks in advance!

**Comment by epage** [comment]
> darling_macro will refuse to compile while building on alpine with a musl target.

Thats strange.  Have you opened an issue with them?

> sass-rs will refuse to compile while building on debian with a musl target.

We have sass configured as a Rust feature.  You could build with no default features and then re-enable the ones you want / work.

**Comment by Elinvynia** [comment]
There is https://github.com/TedDriggs/darling/issues/94, the repo had its last activity 2 months ago. The reason I asked here is in case anyone had some experience trying to compile cobalt with SASS enabled on musl.

I've managed to get it working without the SASS feature using this Dockerfile: https://github.com/Elinvynia/blog/blob/master/Dockerfile but I figured it would be nice to have it enabled.
